showcase/app/templates/components/copy/button.hbs

```diff
@@ -142,6 +142,17 @@
     </SF.Item>
   </Shw::Flex>
 
+  <Shw::Text::H4>Special cases</Shw::Text::H4>
```

Comment:
[suggestion] since both the `Copy::Button` and the `Copy::Snippet` have been technically updated, we should add similar use cases to the `Copy::Snippet` showcase page too

Reply:
Just added.

Reply:
https://hds-showcase-git-hds-4447-copybutton-empty-string-hashicorp.vercel.app/components/copy/snippet#demo

Reply:
Updated again

Reply:
https://hds-showcase-git-hds-4447-copybutton-empty-string-hashicorp.vercel.app/components/copy/snippet#content